src/mysql.php

```diff
@@ -53,11 +55,12 @@ function mysql_connect(
                 $password = ini_get('mysqli.default_pw') ?: null;
             }
 
-            $hash = sha1($hostname . $username . $flags);
+            $hash = sha1($hostname.$username.$flags);
```

Comment:
I much prefer the additional whitespace, are you following a code standard I'm unaware of here? Currently I follow PSR-1/2, did I miss this? 

This applies to all occurrences of this change.

Reply:
Yeap, automatic php-cs-fixer